services/datamanager/builtin/builtin.go

```diff
@@ -217,21 +232,10 @@ func getDurationFromHz(captureFrequencyHz float32) time.Duration {
 // Initialize a collector for the component/method or update it if it has previously been created.
 // Return the component/method metadata which is used as a key in the collectors map.
 func (svc *builtIn) initializeOrUpdateCollector(
+	md componentMethodMetadata,
```

Comment:
[nit]: I think componentMethodMetadata is confusing. It seems like this is just meant to identify a particular collector. In that case, does collectorMetadata make sense?

Reply:
I had switched back and forth about renaming this collectorID - would that be more clear?

Reply:
+1 to collectorID.